README.md

```diff
@@ -13,6 +13,8 @@ Project Milestones:
 * [Code of Conduct](https://github.com/lndk-org/lndk/blob/master/code_of_conduct.md)
 * [Architecture](https://github.com/lndk-org/lndk/blob/master/ARCH.md)
 
+If you have comments or questions about the project feel free to email orbitalturtle@protonmail.com or XXX for an invite to the BOLT 12 Discord. You can find us in the lndk channel.
```

Comment:
I would update this to encourage people to file issues / discussions if they run into issues (because it can be an information sink to answer user questions on a forum like discord) + include the link for "community"

Reply:
I do agree here as it's probably also going to generally be more discoverable. When the project is more mature we can even answer some of the more frequent questions on the Bitcoin Stack Exchange as they come up to make it even more discoverable externally.

Reply:
Good point 👍🏻